assets/app.js

```diff
@@ -21,7 +23,7 @@ $(function() {
   var backgroundColorCounter = 1;
 
   // Homepage intro animation
-  animateHomepageIntro();
+  // animateHomepageIntro();
```

Comment:
If we’re not using it, let’s remove it, including the commented out reference it it above. There will be less to pay attention to when debugging, and if we need it again we can find it in git.